### DISCUSSION TITLE
Toolbar configuration - show only selected items

### DISCUSSION THREAD

**Discussion by ttr42** [keepassxreboot/keepassxc#7842] [Ideas]
Hi,
from time to time I accidentally delete entries just by a click on the wrong icon 
('Delete' is just next to 'Copy Username')

So I was looking, if there is a way, to configure the items, that appear in the toolbar ... so far I didn't find a way to do this.
Does anyone know a way to remove the 'Delete' from the toolbar?

Alternative, could this be some feature for future releases?

Greetings from NRW/Germany,
Thomas

**Comment by droidmonkey** [comment]
Unlikely to be a feature. There is a setting to confirm prior to moving to recycle bin, recommend enabling that.
